Rlog/ViewModel/Schedule/ScheduleListViewModel.swift

```diff
@@ -6,9 +6,26 @@
 //
 
 import SwiftUI
+import Combine
```

Comment:
Framework 의 import를 ABC순으로 작성해두면 추후에 알아보기가 쉽답니다 ✨ 

```suggestion
import Combine
import SwiftUI
```

Reply:
import 정리방법을 알려주셔서 감사합니다! 확인한 결과 `import Combine`이 필요가 없어져서 아예 삭제했습니다...